### ISSUE TITLE
Vendor library name tracking issue

### ISSUE THREAD

**Issue by cpavlina** [cpavlina/kicad-schlib#14]
There's currently a bit of a mix of vendor library names:

- semi-diode-Vendor and semi-trans-Vendor for diodes/transistors
- Vendor for generic parts

In addition, there are libraries for vendors that are now owned by others (for example Atmel, now acquired by Microchip).

I'm not sure of my original decision to split off semi-diode-* and semi-trans-*; maybe these should just be in the main vendor library. Filtering can be done by the component selector.

I think I want to keep separate vendors that are more recognized by an old name - for instance, keep Atmel parts under Atmel. Open to comments on that.

**Comment by iromero91** [comment]
I agree, it is best to keep "historical vendors" separate if they are still recognizable. Huge brands like LT(ADI) or Atmel (MCHP), but perhaps not Supertex (MCHP) or other tiny chip companies that got eaten by the big players. I would ditch the separation of semi-diode, semi-trans, etc and just name every non-generic library with a vendor/brand name with a standardized capitalization.
